coda_v2_python_client/firebase_client_wrapper.py

```diff
@@ -622,3 +622,39 @@ def set_user_ids(self, dataset_id, user_ids):
                 batch.set(self.get_segment_ref(segment_id), {"users": user_ids})
         batch.commit()
         log.debug(f"Wrote {len(user_ids)} users to dataset {dataset_id}")
+
+    def add_message_to_dataset(self, dataset_id, message, MAX_SEGMENT_SIZE=2500):
```

Comment:
MAX_SEGMENT_SIZE here isn't a constant. I think you want this to be
```suggestion
    MAX_SEGMENT_SIZE = 2500

    def add_message_to_dataset(self, dataset_id, message, max_segment_size=MAX_SEGMENT_SIZE):
```